bokehjs/src/lib/core/util/svg.ts

```diff
@@ -1068,8 +1068,13 @@ export class SVGRenderingContext2D implements BaseCanvasRenderingContext2D {
     start_angle = start_angle % (2 * Math.PI)
     end_angle = end_angle % (2 * Math.PI)
 
-    const start_x = x + radius_x * Math.cos(start_angle)
-    const start_y = y + radius_y * Math.sin(start_angle)
+    const cos_rotation = Math.cos(rotation)
+    const sin_rotation = Math.sin(rotation)
+
+    const dx = radius_x*Math.cos(start_angle)
+    const dy = radius_y*Math.sin(start_angle)
+    const start_x = x + dx*cos_rotation - dy*sin_rotation
+    const start_y = y + dx*sin_rotation + dy*cos_rotation
```

Comment:
This and other instances could use `AffineTransform()` to do the rotation.

Reply:
Done.